openedx_tagging/core/tagging/api.py

```diff
@@ -484,3 +484,60 @@ def delete_tags_from_taxonomy(
     """
     taxonomy = taxonomy.cast()
     taxonomy.delete_tags(tags, with_subtags)
+
+
+def copy_tags(
+    source_object_id: str,
+    dest_object_id: str,
+    include_deleted: bool = True,
+    object_tag_class: type[ObjectTag] = ObjectTag
+):
+    """
+    Copy all tags from one object to another.
+
+    This keeps all not-copied tags and delete all
+    previous copied tags of the dest object.
+    If there are not-copied tags that also are in 'source_object_id',
+    then they become copied.
+    """
+    ObjectTagClass = object_tag_class
+
+    def dest_object_has_tag(object_tag):
+        try:
+            result = ObjectTagClass.objects.get(
+                object_id=dest_object_id,
+                _export_id=object_tag.export_id,
+                _value=object_tag.value,
```

Comment:
I think it's better to match using `taxonomy_id` and `tag_id` rather than `_export_id` and `_value`, assuming that we don't need to support deleted tags. See below for my recommendation to use `update_or_create`

Reply:
Updated: https://github.com/openedx/openedx-learning/pull/236/commits/744a539b9a1abfd79a698943e12468de029cbc9f